src/handlers/v1/markets/[chainId]/[marketId]/collateral-swap/index.ts

```diff
@@ -134,17 +134,17 @@ export const v1GetCollateralSwapQuotationRoute: Route<GetCollateralSwapQuotation
       approvals = estimateResult.approvals;
 
       // 7. calc target position
-      const withdrawalUSD = new BigNumberJS(amount).times(withdrawalCollateral.assetPrice);
-      const targetUSD = new BigNumberJS(targetTokenAmount).times(targetCollateral.assetPrice);
+      const withdrawalUSD = new BigNumberJS(srcAmount).times(srcCollateral.assetPrice);
+      const targetUSD = new BigNumberJS(destAmount).times(destCollateral.assetPrice);
```

Comment:
Do you think those naming should follow `src` and `dest` also? Like `srcUSD` and `destUSD`. Or we can follow the position by using `current` and `target`.

Reply:
I think it's best to use `targetXXX` to describe the fields of `targetPosition`. For `withdrawalUSD`, I prefer using the action since `srcUSD` is more difficult to understand.